version-stable-r-development.Rmd

```diff
@@ -83,38 +91,42 @@ following setup:
 - No authentication required (local setup).
 - Enable root by setting the environment variable `ROOT` to `TRUE`, so that e.g.
 `sudo apt-get` can be used in RStudio.
-- Use a version-specific port, e.g. `3500` for R 3.5.0, `3610` for R 3.6.1 and
-so on, so that we can use `localhost` for concurrent R version instances.
+- Use a version-specific port, e.g. `4000` for R 4.0.0, `4410` for R 4.4.1 and
+so on, so that we can use `localhost` for concurrent R version instances. 
+We bind the port to localhost (`127.0.0.1:4410`), so it is only accessible locally 
+(see [the Rocker reference](https://rocker-project.org/images/versioned/rstudio.html#disable_auth)).
 - The development code of all relevant projects should live outside the
-container and be shared with it (and possibly many of them), e.g. under
-`~/RStudioProjects` on the host machine and `/home/rstudio/RStudioProjects` in
+container and be shared with it (and possibly multiple other containers), e.g. under
+`~/workspace` on the host machine and `/home/rstudio/workspace` in
 the container.
     - For this to work w/o [permission
     issues](https://github.com/rocker-org/rocker/wiki/Sharing-files-with-host-machine#avoiding-permission-changes-when-sharing-volumes),
-    the container user (`rstudio`) must match the UID of the host user (`$UID`).
-- In order for the RStudio setting to persist if the container is recreated
+    the container user (`rstudio`) must match the UID of the host user (`$UID`). 
+    This has the effect of setting the ownership of `~/workspace` on the host machine to `$UID` if it is not already owned by that user.
```

Comment:
The link should be updated to https://rocker-project.org/images/versioned/rstudio.html#userid-and-groupid instead of the legacy repo wiki

Also, maybe we should align to what is documented there and also set `GID` via `-e GROUPID`

Reply:
done